src/app/pages/dashboard/components/go-to-journal-card/go-to-journal-card.html

```diff
@@ -1,12 +1,12 @@
 <button class="go-to-journal-button" (click)="navigateToJournal()">
-  <ion-card class="go-to-journal-card">
+  <ion-card class="go-to-journal-card" id="go-to-journal-card">
     <div id="card-content-wrapper">
       <ion-grid>
         <ion-row class="icon-row">
           <img src="assets/imgs/dashboard/go-to-journal.png" alt="Journal" class="icon-journal"/>
         </ion-row>
         <ion-row>
-          <ion-button mode="ios" class="mes-primary-button">
+          <ion-button id="go-to-journal" mode="ios" class="mes-primary-button">
```

Comment:
call this "go-to-journal-button" instead of just "go-to-journal"

Reply:
done